src/context/LocationsContext.js

```diff
@@ -7,11 +7,20 @@ export const LocationsContext = createContext();
 const Locations = ({ children }) => {
   const valueMapCenter = useContext(MapCenterContext);
   const { newCenterMap } = valueMapCenter;
-  const lat = newCenterMap.lat;
-  const lng = newCenterMap.lng;
-  const { isFetching, locations, status, error } = useLocations(lat, lng);
-  const value = { locations, isFetching, status, error };
+  const [wait, setWait] = useState(0);
+  const { isFetching, locations, status, error } = useLocations(
+    newCenterMap,
+    wait * 1000,
```

Comment:
**Optional Suggestion:** What do you think if we use a variable `delayInSeconds = wait * 1000`?;

Reply:
I think we’re good, and this bit of code looks fine

Reply:
I like Alejandro’s suggestion here. Again, it’s about making the code easier for your teammates (and future you!) to understand.

Reply:
Done, we now have a separate  variable for storing the value